servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/InfluencerConnectionAcceptor.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.netty.internal;
+
+import io.servicetalk.concurrent.api.Completable;
+import io.servicetalk.transport.api.ConnectExecutionStrategy;
+import io.servicetalk.transport.api.ConnectionAcceptor;
+import io.servicetalk.transport.api.ConnectionAcceptorFactory;
+import io.servicetalk.transport.api.DelegatingConnectionAcceptor;
+import io.servicetalk.transport.api.ExecutionStrategyInfluencer;
+
+import static io.servicetalk.concurrent.api.Completable.completed;
+import static io.servicetalk.transport.api.ConnectExecutionStrategy.anyStrategy;
+
+/**
+ * A contract that defines the connection acceptance criterion.
+ *
+ * @see ConnectionAcceptorFactory
+ */
+@FunctionalInterface
+public interface InfluencerConnectionAcceptor extends ConnectionAcceptor,
```

Comment:
Any reason we can't use the same pattern established with `HttpExecutionStrategyInfluencer` (an influencer interface, and the filter/factory extends the influencer)?

suggested:
```java
// rename InfluencerConnectionAcceptor to ConnectExecutionStrategyInfluencer
interface ConnectExecutionStrategyInfluencer extends ExecutionStrategyInfluencer<ConnectExecutionStrategy> {}


interface ConnectionAcceptor extends ConnectExecutionStrategyInfluencer {}
```

Reply:
discussed offline. 👍 my misunderstanding on how this was intended to be used. this interface is internal only.